andela/haku-cfh
# Feature/146683105/redesign game user interface

#### What does this PR do?
This PR is redesigning the user interface of the gaming screen

#### Description of Task to be completed?
Change the existing user interface and give the gaming screen a new look.

#### How should this be manually tested?
Sign in as a user and click on the "Play Game" button to gain access to the gaming screen to check out the new user interface. 

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
Re-Design UI for the actual gaming screen

#### Screenshots (if appropriate)
<img width="1279" alt="screen shot 2017-07-18 at 5 18 06 pm" src="https://user-images.githubusercontent.com/10426118/28328121-5c049528-6bdd-11e7-9500-178478136b58.png">

<img width="1279" alt="screen shot 2017-07-18 at 5 19 12 pm" src="https://user-images.githubusercontent.com/10426118/28328147-69aae650-6bdd-11e7-9259-b2bdece2aa83.png">


#### Questions:
N/A
